hack/build/docker/cdi-operator/Dockerfile

```diff
@@ -17,6 +17,5 @@ WORKDIR /home/cdi-operator
 USER 1001
 
 COPY ./cdi-operator /usr/bin/cdi-operator
-COPY ./csv-generator /usr/bin/csv-generator
```

Comment:
Should this stay?

Reply:
Same answer, if we don't care about upstream CSV generator, then we don't need to compile and install it in the container.

Reply:
Only reason I can see it remaining here, is to stop the Dockerfile check downstream from triggering that something changed.